src/cuda_utils.cc

```diff
@@ -83,39 +83,41 @@ GetDeviceMemoryInfo(const int device_id, size_t* free, size_t* total)
 }
 
 Status
-EnablePeerAccess(const double min_compute_capability)
+EnablePeerAccess(const double min_compute_capability, bool enable_peer_access)
```

Comment:
seems odd to have a function that has a bool that to disable it?

Why wouldn't we guard against calling this function instead?